src/tracer/MOM_offline_control.F90

```diff
@@ -0,0 +1,619 @@
+!***********************************************************************
```

Comment:
Boiler plate should be just one line. Comments in doxygen: https://github.com/NOAA-GFDL/MOM6/wiki/Doxygen#header